REQUIRE

```diff
@@ -1,3 +1,3 @@
 julia 0.4
-Compat 0.8.0
+Compat 0.9.5
```

Comment:
I think maybe an earlier version tried to use a buffer and `String(take!(buf))` for the `@btime` test, which requires Compat 0.9.5.   I then rewrote it to use `redirect_stdout(f)` with a temporary file, which requires Compat 0.9.0 (https://github.com/JuliaLang/Compat.jl/commit/2c11855a34bd2a9e277f5ddbec8eed49df48a02b)